pkg/apiserver/apiserver.go

```diff
@@ -40,7 +40,9 @@ var (
 	Scheme = runtime.NewScheme()
 	// Codecs provides methods for retrieving codecs and serializers for specific
 	// versions and content types.
-	Codecs = serializer.NewCodecFactory(Scheme)
+	Codecs          = serializer.NewCodecFactory(Scheme)
+	TokenPath       = "/tmp/controller-loopback-token"
```

Comment:
Maybe use some more formal path like `/var/run/antrea/apiserver/loopback-client-token`

Reply:
I thought this would export the token file to the host since this is under a mounted dir. Updated anyway, and I would like to learn from you more.

Reply:
Even it's written to /tmp, it's also readable from the host by root user as the filesystem of the container is always on the host. So there's no real security difference between /tmp and /var/run/antrea. But I don't think we need to prevent root user to access the token file. Other users cannot read the file as long as you set it to 0600.

Reply:
Got it, changed perm to 0600 also.